components/migrate/steps/SignLoader1.vue

```diff
@@ -24,11 +24,9 @@
     </div>
     <div class="flex">
       <div class="v-border"></div>
-      <div class="mb-4 flex">
+      <div class="mb-4 w-full flex">
         <div class="mr-4">
-          <NeoIcon v-if="step1Iterations === 0" v-bind="iconSuccess" />
-          <NeoIcon v-else-if="step1Iterations === 1" v-bind="iconLoading" />
-          <NeoIcon v-else v-bind="iconIdle" />
+          <NeoIcon v-bind="whichIcon()" />
```

Comment:
what's this? looks new

Reply:
I decide to change the code similar with line 5 https://github.com/kodadot/nft-gallery/blob/80e9d3fb5b44d3f97af92b50216c7e5680100b84/components/migrate/steps/SignLoader1.vue#L5

Reply:
ain't it better to have a computed property? just asking, I'm not really familiar with `v-bind`

Reply:
oh, `v-bind` here is to apply the attributes to the component. 

```vue
<NeoIcon v-bind="whichIcon()" />
```

for `iconIdle`, will transform to
```vue
<NeoIcon icon="circle" class="text-k-grey-light" pack="fass" />
```

Reply:
aaaaaah now I remember thank you! that what I was looking last time